src/track/trackmetadatataglib.cpp

```diff
@@ -1867,7 +1867,7 @@ bool exportTrackMetadataIntoID3v2Tag(TagLib::ID3v2::Tag* pTag,
     DEBUG_ASSERT(pHeader);
     if (!checkID3v2HeaderVersionSupported(*pHeader)) {
         kLogger.warning() << "Legacy ID3v2 version - exporting only basic tags";
-        exportTrackMetadataIntoTag(pTag, trackMetadata);
+        exportTrackMetadataIntoTag(pTag, trackMetadata, WRITE_TAG_OMIT_NONE);
```

Comment:
unrelated change?

Reply:
Yep, totally unlerated but building failed without this. Occurred since one master merge from @uklotzde.

Reply:
Ah, I see this change is already part of current master. 
Can you rebase your commit on top of the current master: 
"git rebase upstream/master" 
This way we avoid to track the same change in two git branches.

Reply:
Worked perfectly.